src/styles/header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $header-bg;
+  width: calc(100% - #{$header-padding * 2});
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $logo-size;
```

Comment:
The variable `$logo-size` is used here, but it is not defined within this file. Ensure that `$logo-size` is defined in a file that is imported into this one, or define it here if it's missing.